src/backend/app/Http/Middleware/TrustProxies.php

```diff
@@ -11,14 +11,18 @@ class TrustProxies extends Middleware {
      *
      * @var array<int, string>|string|null
      */
-    protected $proxies;
+    protected $proxies = [
+        '127.0.0.1',
+        '::1',
+        // Add your production proxy IPs here
+        // For example: '10.0.0.0/8', '172.16.0.0/12', '192.168.0.0/16'
+    ];
```

Comment:
I don't think this does anything. From what I understand in our set up the Proxy IP's are the ones from GCE LoadBalancers: https://cloud.google.com/load-balancing/docs/https#global-classic-connections

However, I don't think we want to "hardcode" GFE here, else it will be hard for people to re-use the code. Looking around, it seems the Middleware internally uses `trustedproxy.proxies`

```php
$trustedIps = $this->proxies() ?: config('trustedproxy.proxies');
```

So, maybe we can just add a `config` that reads it from ENV.
Example can be found here: https://github.com/laravel/docs/pull/7457/

And then, we can add the GFE IP ranges to `configmap-cloud|demo-backend.yaml`. And also add it to deployment guide, as this is not trivial (I think).

Reply:
Thanks for the explanation, I will be looking in this.

Reply:
Thanks for the explanation, I will be looking in this.